mmocr/models/ner/convertors/ner_convertor.py

```diff
@@ -0,0 +1,174 @@
+from mmocr.models.builder import CONVERTORS
+
+
+@CONVERTORS.register_module()
+class NerConvertor:
```

Comment:
Be careful to use the keyword `Ner`, unless all the ner methods using the same convertor.